package.json

```diff
@@ -28,7 +28,7 @@
     "acorn": "^2.7.0",
     "alpheios-data-models": "github:alpheios-project/data-models",
     "alpheios-experience": "github:alpheios-project/experience",
-    "alpheios-inflection-tables": "github:alpheios-project/inflection-tables#inflection-tables-vue",
+    "alpheios-inflection-tables": "github:alpheios-project/inflection-tables#infltbl-i109",
```

Comment:
May be it is better to remove the branch before merging to master?

Reply:
Agree, will remove it before merging, thanks for catching this!